### ISSUE TITLE
Uses footstep sounds different from the original mod for vanilla armors

### ISSUE THREAD

**Issue by lilebonymace** [tr4wzified/AOSISCSoundPatcher#12]
![изображение](https://user-images.githubusercontent.com/65489649/202987628-489027bf-13b5-4572-9ac2-70f86389f7e9.png)

**Comment by tr4wzified** [comment]
Are you sure you didn't run Synthesis and then activated this floating weapons fix ESP afterwards?

**Comment by lilebonymace** [comment]
Yes i'm sure. In fact this result would look even more weird if i did,
